CONTRIBUTING.md

```diff
@@ -100,74 +174,39 @@ the following categories:
 > `pnpm changeset add --empty` to generate an empty changeset file to document
 > your changes.
 
-## Git Hooks
+# Git Hooks
 
 The SDK utilizes a pre-push git hook to validate your contribution before review. This is a script that will run automatically before changes are pushed to the remote repository. Within the SDK, the pre-push script will run code linting.
 
 > This can be overridden using the `--no-verify` flag when pushing.
 
-## Build and watch all packages
-
-If you want to work locally using realtime builds, open in one terminal tab build in watch mode
-on all packages from the root directory:
-
-```sh
-pnpm build:watch
-```
-
-This command will run `tsup --watch` on all packages using Turborepo
-
-## Using linked packages
-
-This will link all packages inside our monorepo in your `global pnpm store`, enabling you to use `fuels-ts` packages via links in
-your local projects.
-
-### On `fuels-ts` root directory
-
-```sh
-pnpm -r exec pnpm link --global --dir ./
-```
-
-You can use [build watch](#build-and-watch-all-packages).
-
-### On `your project` root directory
-
-```sh
-pnpm link --global fuels
-```
+# Updating Forc version
 
-Or for specfic packages just use `pnpm link @fuel-ts/<pkg-name>`, ex;
+The following script will upgrade Forc to the latest version on GitHub, remove all lockfiles so the latest stdlib can be used, and rebuild all projects:
 
 ```sh
-pnpm link --global @fuel-ts/wallet
+pnpm forc:update
 ```
 
-### Troubleshooting
-
-If you're linking for the first time, you might need:
+After this you should run tests and fix any incompatibilities.
 
-```sh
-  pnpm setup
-```
+# Updating Fuel Core version
 
-If it still have problems, you might need to setup again (As `pnpm` releases new version, the global folder structure may change)
+Manually edit the `services/fuel-core/Dockerfile`, add the right version, and then:
 
 ```sh
-  pnpm setup
+pnpm services:clean # causes rebuilding of the Docker image
+pnpm test:ci
```

Comment:
Should we just add `pnpm services:clean` to the start of the script in `scripts/ci-test.sh` that `pnpm test:ci` is executing?